R/usage.R

```diff
@@ -110,7 +110,7 @@ fun_info <- function(fun) {
       list(
         type = "s4",
         name = as.character(x[[2]]),
-        signature = purrr::map_chr(as.list(x[[3]][-1]), as.character)
+        signature = as.character(purrr::pluck(x[[3]][-1]))
```

Comment:
This doesn’t look right to me because pluck should have two arguments.

Reply:
You're right. This works, but not for the right reason.